src/UrlParams.ts

```diff
@@ -408,6 +405,22 @@ export const getUrlParams = (
       };
   }
 
+  const intentDerivedConfiguration: IntentDerivedConfiguration = {};
+  const desktopMobile = platform === "desktop" ? "desktop" : "mobile";
```

Comment:
Desktop was confusing me because on web we have separate platforms for "web" and "desktop". I assume this is effectively "mobile" or "other".

Reply:
This has been confirmed oob that this is how it works.

Reply:
added a comment.